packages/components/src/components/primitives/Link/types.ts

```diff
@@ -0,0 +1,30 @@
+import { LinkProps as ChakraLinkProps } from '@chakra-ui/react';
```

Comment:
Could we be consistent and try to adapt this to follow the implementation of the `Button` for example? Instead of `interfaces` use `consts` etc... Check `Button` types for references 👍🏻